### ISSUE TITLE
gregtech galaxyspace problem

### ISSUE THREAD

**Issue by adamcirillo** [BlesseNtumble/GalaxySpace#181]
I seem to be having an issue with adding galaxyspace to a pack I'm making where once I add it the gregtech recipe for the implosion compressor stops working. in 1.1.7 galaxyspace I would click the recipe for compressed steel and would take me to the recipe for steel plate. This change with 1.1.8 where it shows the recipe as 2 steel plate + 1 tnt and outputs 1 compressed steel + 1 tiny pile of ashes, when I put the items in the machine it takes 1 steel plate and 1 tnt and outputs 1 tiny pile of ashes no compressed steel. This issue goes away if I remove galaxy space from the modpack. I also tested standalone with just gregtech, galacticraft and galaxyspace just to make sure no other mod was causing the issue.

mod versions
gregtech 5.09.31
galaticraft 3.0.12.502
galaxyspace 1.1.18

here is the issue I've opened up on gregtech although I think it might be on your end
Blood-Asp/GT5-Unofficial#1224

**Comment by BlesseNtumble** [comment]
The recipe for compressed plates from GC3 I did not change.

**Comment by adamcirillo** [comment]
well it only happens when I add your mod if its not there the recipe works as intended I haven't heard a reply back from the gt devs about it yet.

**Comment by adamcirillo** [comment]
I heard back from one of the gt devs, he said the issues is related to how you change the oredict. this is what he said.

 I looked at the GT code in question and I don't think that this is a GT issue.
All the code in question does is add recipes for Plate + Explosive -> Compressed + Dark Ash.
As Alkalus pointed out, the problem seems to be lying with OreDict registration.
If Galaxy Space is installed the GC compressed plates are registered as plateSteel instead of compressedSteel.
(Copper, Bronze, etc. had the same issue when I tested it.)
I don't think I will be able to help you unless having GT installed for some reason influences how GC compressed plates are being registered.
